templates/keepalived.conf.j2

```diff
@@ -13,6 +13,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+global_defs {
+  enable_script_security
+}
```

Comment:
I think this should not be part of the template by default, instead we should a loop over any global_defs, and write their value.

so we should probably have something like:
```
{% if keepalived_global_defs is defined %}
{% for def in keepalived_global_defs.items() %}
{{ def }}
{% endfor %}
{% endif %}
```

This we can pass a variable, whose content would be:
```
keepalived_global_defs:
  - enable_script_security
```

This way global notification scripts can work too.